code/modules/recycling/disposal.dm

```diff
@@ -152,6 +152,8 @@
 		target.visible_message("<span class='danger'>[user] starts putting [target] into [src].</span>", \
 								"<span class='userdanger'>[user] starts putting you into [src]!</span>")
 	if(do_mob(user, target, 20))
+		if (!src.loc)
```

Comment:
Why is that bad? Varedit shows the chute continues to exist if you break it while going in, but its loc gets set to null, so this works